.github/workflows/release.yml

```
@@ -26,7 +26,7 @@ jobs:
         # an output. It _does_ register that token as a secret so that it will be
         # filtered from log output automatically
         id: generate-token
-        uses: tibdex/github-app-token@586e1a624db6a5a4ac2c53daeeded60c5e3d50fe
+        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06
```

Here (and everywhere), can we add a comment that explains what version this maps to? We should have done that before...

```suggestion
        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06 # v1.6.0
```

Also, feel free to tell me that we should consider just using the `release/v1.6` branch. We had that discussion on the PR where this was originally introduced so before starting a convo, please read the earlier one for context.